libnopegl/src/node_renderother.c

```diff
@@ -1043,8 +1051,8 @@ static int rendertexture_prepare(struct ngl_node *node)
         {.name = "tex_coord", .type = NGLI_TYPE_VEC2},
     };
 
-    const struct pipeline_desc *descs = ngli_darray_data(&c->pipeline_descs);
-    const struct pipeline_desc *desc = &descs[ctx->rnode_pos->id];
+    struct pipeline_desc *descs = ngli_darray_data(&c->pipeline_descs);
+    struct pipeline_desc *desc = &descs[ctx->rnode_pos->id];
```

Comment:
you don't need to change these (`rendertexture`)

Reply:
Fixed.